### PR TITLE
cuda: New version 10.1.243

### PR DESCRIPTION
**Pull request by jabl**
Adds CUDA 10.1.243. The installer command-line arguments have slightly
changed, hence in the install method the version needs to be inspected
in order to give the correct arguments for the new installer.

### CODE REVIEW

File: var/spack/repos/builtin/packages/cuda/package.py [hunk 1]
```diff
@@ -64,13 +68,22 @@ def install(self, spec, prefix):
...
+            runfile(
+                '--silent',         # disable interactive prompts
+                '--override',       # override compiler version checks
+                '--toolkit',        # install CUDA Toolkit
+                '--installpath=%s' % prefix
+            )
+        else:
+            runfile(
+                '--silent',         # disable interactive prompts
+                '--verbose',        # create verbose log file
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/cuda/package.py]: Can you gather together the common arguments? Something like:
```python
arguments = [
    '--silent',         # disable interactive prompts
    '--override',       # override compiler version checks
    '--toolkit',        # install CUDA Toolkit
]
if spec.satisfies('@10.1:'):
    arguments.append('--installpath=%s' % prefix)
else:
    arguments.append('--verbose')
    arguments.append('--toolkitpath=%s' % prefix)

runfile(*arguments)
```

Comment (jabl) [reply var/spack/repos/builtin/packages/cuda/package.py]: Sure, done in the follow-up commit.

Comment (alalazo) [reply var/spack/repos/builtin/packages/cuda/package.py]: `flake8` not happy :slightly_smiling_face: 

> var/spack/repos/builtin/packages/cuda/package.py:80: [E501] line too long (85 > 79 characters)

Comment (jabl) [reply var/spack/repos/builtin/packages/cuda/package.py]: Agh, fixed..

Comment (alalazo) [reply var/spack/repos/builtin/packages/cuda/package.py]: No worries, waiting for @ax3l to give his review - if you don't hear anybody back in a couple of days ping me please!

**Pull request merged**